force-app/main/default/objects/Trailer__c/fields/Trailer_URL__c.field-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Trailer_URL__c</fullName>
+    <externalId>false</externalId>
+    <label>Trailer URL</label>
+    <required>false</required>
```

Comment:
I think it should be required. Trailed without url has no sense.

Reply:
It's true